docs/form-customization.md

```diff
@@ -76,6 +76,28 @@ render((
 
 Here's a list of supported alternative widgets for different JSONSchema data types:
 
+#### Creating a `select` element
+
+To create a select element, you can use `enum`. The array elements will render as options. To add labels that differ from the option's values use `enumNames`:
+
+```js
+const schema = {
+  type: "string",
+  enum: ['AU', 'FR', 'GB'],
+  enumNames: ["Australia", "France", "United Kingdom"]
```

Comment:
Could you change all instances of "United Kingdom" to "Great Britain", or change "GB" to "UK", just for consistency?